### DISCUSSION TITLE
CRIT_DISK_USAGE setting in SO 2.3

### DISCUSSION THREAD

**Discussion by tsmullins** [Security-Onion-Solutions/securityonion#4052] [General]
Good afternoon,

We need to modify the percentage of disk usage when the NSM scripts should start purging old data. In the old version, this setting was in:
/etc/nsm/securityonion.conf
CRIT_DISK_USAGE=90

Could someone share the location of the new setting?

Thanks 
Shane

**Comment by weslambert** [comment]
We don't currently expose this in a pillar, but for the time being, you could copy the utility to the equivalent `/opt/so/saltstack/local` directory and make your changes there.

**Reply by tsmullins** [reply]
Many thanks Wes.
